kubernetes/GKE/README.md

```diff
@@ -66,7 +81,15 @@ git clone https://github.com/pytorch/serve.git
 cd serve/kubernetes/GKE
 ```
 
-#### 2.2 Install NVIDIA device plugin
+**_NOTE:_** By default the helm chart installs GPU version of torchserve. Follow steps in section [2.2](####-2.2-For-CPU-setup) for running in a CPU only cluster. For GPU setup section [2.2](####-2.2-For-CPU-setup) can be skipped.
+
+#### 2.2 For CPU setup
+
+* Change torchserve image in Helm/values.yaml to the CPU version
+* Set `n_gpu` to `0` in Helm/values.yaml
+* Skip NVIDIA plugin installation in section [2.3](#####-2.3-Install-NVIDIA-device-plugin)
```

Comment:
@jagadeeshi2i In addition to these, when the cluster is created one will use CPU node pool. Mention that for the cluster creation